### DISCUSSION TITLE
Vtx & Vrx

### DISCUSSION THREAD

**Discussion by Lalbabu2001** [ExpressLRS/ExpressLRS#2877] [Ideas]
As we pushed radio to groundbreaking levels, using affordable hardware, can we do same for video transmission system, can well developed something like express elrs for video.

**Comment by mha1** [comment]
Something like this? https://github.com/OpenHD/OpenHD

**Reply by sunjialinsun** [reply]
or OpenIPC https://github.com/OpenIPC   , as there is HW for openIPC on FPV .
